resources/views/components/content-area.blade.php

```diff
@@ -16,7 +16,7 @@
                     <div class="offcanvas-main-menu mt:hidden">
                         <ul>
                             <li>
-                                <a class="main-menu-toggle">Main Menu</a>
+                                <a class="main-menu-toggle" tabindex="0" aria-expanded="false">Main Menu</a>
```

Comment:
I had to put tabindex="0" in order to be able to tab to this link. I am not sure why the DOM didn't naturally pick it up, but this fixes the current issue of not even being able to tab to this element.